crates/node_binding/binding.d.ts

```diff
@@ -136,7 +136,7 @@ export class JsResolver {
 
 export class JsResolverFactory {
   constructor()
-  get(type: string, options?: RawResolveOptionsWithDependencyType): JsResolver
+  get(type: string, options?: RawResolveOptionsWithDependencyType,pnpManifestPath: string|undefined): JsResolver
```

Comment:
Do you plan to move the `pnpManifestPath` into the option bag (here and in the other places where the manifest path is passed as free function)? Ideally the PnP-specific references should be kept to a minimum imo.

Reply:
no, don't want to change resolveOption api signature, I'm not sure whether it's worthy to support manifest per request or manifest per project. if we need to support manifest per project, we need to pass it to resolveOptions at least in rust side.